### PR TITLE
Add support for local tar file in read_entityset

### LINKED ISSUE #1227: Entityset deserialization does not support local tar file
### Entityset deserialization does not support local tar file
- `read_entityset` supports reading an EntitySet in a tar file from an S3 Link, URL. 
- However, **it does not support reading an EntitySet in a tar file from a local directory**.

```python
from featuretools import read_entityset
from urllib.request import urlretrieve

TEST_FILE = "test_serialization_data_entityset_schema_4.0.0.tar"
URL = "https://featuretools-static.s3.amazonaws.com/" + TEST_FILE

# download from Link and save at current directory
local_folder_filepath = "test_serialization_data_entityset_schema.tar"
urlretrieve(URL, filename=local_folder_filepath)

link_es = read_entityset(URL)
local_es = read_entityset(local_folder_filepath)

assert local_es.__eq__(link_es, deep=True)
```

#### Stacktrace
```python
---------------------------------------------------------------------------
NotADirectoryError                        Traceback (most recent call last)
<ipython-input-5-c4cb724b9dce> in <module>
     10
     11 link_es = read_entityset(URL)
---> 12 local_es = read_entityset(local_folder_filepath)
     13
     14 assert local_es.__eq__(link_es, deep=True)

~/featuretools/featuretools/entityset/deserialize.py in read_entityset(path, profile_name, **kwargs)
    213             return description_to_entityset(data_description, **kwargs)
    214     else:
--> 215         data_description = read_data_description(path)
    216         return description_to_entityset(data_description, **kwargs)

~/featuretools/featuretools/entityset/deserialize.py in read_data_description(path)
    182     assert os.path.exists(path), '"{}" does not exist'.format(path)
    183     file = os.path.join(path, 'data_description.json')
--> 184     with open(file, 'r') as file:
    185         description = json.load(file)
    186     description['path'] = path

NotADirectoryError: [Errno 20] Not a directory: '/Users/gaurav.sheni/test_serialization_data_entityset_schema/data_description.json'
```

### PR DESCRIPTION
**Pull request by gsheni**
- Fixes #1227

### CODE REVIEW

File: featuretools/utils/wrangle.py [hunk 1]
```diff
@@ -168,3 +169,11 @@ def _is_url(string):
     Returns a boolean.
     '''
     return 'http' in string
```
Comment (gsheni) [inline featuretools/utils/wrangle.py:171]: Not a big deal but, should we change the `_is_s3` to be (same for `_is_url`)
```python
def _is_s3(string):	
    '''	
    Checks if the given string is an url path.	
    Returns a boolean.	
    '''	
return string.startswith("s3://")
```

Comment (rwedge) [reply featuretools/utils/wrangle.py:171]: for is url are you thinking `string.startswith("http")` ?

Comment (gsheni) [reply featuretools/utils/wrangle.py:171]: Yes.

Comment (rwedge) [reply featuretools/utils/wrangle.py:171]: Sounds good to me

**Review by tuethan1999 — APPROVED** [review]
Looks good!

**Pull request merged**